scripts/publish_release.sh

```diff
@@ -11,15 +11,15 @@ sha="$(git rev-parse HEAD)"
 
 npx lerna publish from-package --yes
 
-curl -s -X POST https://api.github.com/repos/$REPOSITORY/releases \
--H "Authorization: token $GITHUB_TOKEN" \
--d @- <<EOF
-{
-  "tag_name": "$version",
-  "target_commitish": "$sha",
-  "name": "$version",
-  "body": "Automated release for $version\n",
-  "draft": false,
-  "prelease": false
-}
-EOF
+# curl -s -X POST https://api.github.com/repos/$REPOSITORY/releases \
```

Comment:
If we're leaving this here for now, can you add a comment explaining the problem?

Reply:
I can certainly also remove it, but my intention is to remove it with https://github.com/Vertexvis/vertex-web-sdk/pull/587 immediately after this publishes. This is hopefully just a temporary 15 minute (assuming all goes well) state while waiting for it to publish

Reply:
Sounds good 👍